### PR TITLE
cdep: update-default posting to slack

### PR DESCRIPTION
**Pull request by razvanmuscalu**

### CODE REVIEW

**Comment by billinghamj** [comment]
Is there a way this could be done a little more generically, or is that just the nature of the tool's current design?

**Comment by razvanmuscalu** [comment]
That's just how the current design is. Everything is copy pasted a few times for update, update-web, update-default.

I could put the posting-to-SNS code inside the app function in a common function in a separate file and re-use it though.

**Comment by razvanmuscalu** [comment]
@billinghamj done

File: tools/cdep/commands/update_default.go [hunk 1]
```diff
@@ -5,6 +5,13 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+	"github.com/cuvva/cuvva-public-go/lib/config"
+	"github.com/cuvva/cuvva-public-go/lib/ptr"
+	"github.com/pkg/errors"
+
```
Comment (billinghamj) [inline tools/cdep/commands/update_default.go:14]: No newline here I don't think :) All the GitHub imports get sorted together - including the "local" ones

**Review by billinghamj — APPROVED** [review]
Nicely done! Just worth addressing the import thing

**Pull request merged**